internal/policies/mount/mount.go

```diff
@@ -1,7 +1,16 @@
-// Package mount implements the manager responsible to handle the file sharing
-// policy of adsys, parsing the GPO rules and setting up the mount process for
-// the requested drives. User mounts will be handled by a systemd user service and
-// computer mounts will be handled directly by systemd, via mount units.
+// Package mount provides the policy manager to handle file sharing policies.
+//
+// The manager behavior differs depending on the object type.
+// For system mounts, it will parse the requested values and create systemd mount units to mount the
+// requested shared locations.
+// For user mounts, it will parse the requested values, store them in a separate file which will be
+// handled by a helper binary that will mount the requested shared locations using gio.
+//
+// Should the manager fail to setup the policy with the requested entries and its values,
+// an error will be returned and the login is prevented.
+// However, if the manager creates the files needed and setup all the required steps, it's up to
+// the correctness of the specified entries values and gvfs to mount the requested shared drives.
+// Should an error occur during this step, ADSys will log it without preventing the authentication.
```

Comment:
Well, it’s rather through systemd (mount units for system or our excellent Rust binary for users) logs that our admin will access it? (so that she/he doesn’t look at the adsys logs and don’t see it).